Thai/urbex_bkk

```diff
@@ -0,0 +1,10 @@
+Have you ever explored a deserted building in Bangkok ?
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.